ampersand-multifield-view.js

```diff
@@ -138,12 +181,36 @@ var MultiFieldView = View.extend({
     View.prototype.remove.apply(this, arguments);
   },
 
+  reset: function() {
+    this.setValue(this.startingValue, true);
+    this.fields.forEach(function(field) {
+      if(field.reset) {
+        field.reset();
+      }
+    });
+  },
+
+  clear: function() {
+    // This won't unset field values, but will disable validation
+    this.setValue({}, true);
```

Comment:
if this view actually had its own `value` to set [distinct from its childrens' values] i'd say that makes sense, but as is it's just masking the invocation's true purpose(evidenced by the comment). I'm voting for direct `shouldValidate` manipulation over similar function appearance.